automerge-backend/src/actor_states.rs

```diff
@@ -0,0 +1,95 @@
+use crate::error::AutomergeError;
+use crate::protocol::{ActorID, Change, Clock};
+use std::collections::HashMap;
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct ActorStates {
```

Comment:
Can we add a small doc comment describing what the responsibilities of this struct are?

Reply:
Added